ckanext/unhcr/templates/kobo/surveys.html

```diff
@@ -53,13 +53,21 @@ <h1>My KoBoToolbox Surveys</h1>
             <button
                 class="btn btn-primary kobo-pkg-update-resources"
                 data-kobo-asset-id="{{ survey.uid }}"
+                data-force="false"
                 data-kobo-update-endpoint="{% url_for 'unhcr_kobo.enqueue_survey_update' %}"
-                ><i class="fa fa-refresh"></i> Update KoBoToolbox data
+                ><i class="fa fa-refresh"></i> <span class="update-kobo-button-text">Update KoBoToolbox data</span>
```

Comment:
I think even though the full name is KoboToolbox is fine to just use "KoBo" data

```suggestion
                ><i class="fa fa-refresh"></i> <span class="update-kobo-button-text">Update KoBo data</span>
```

This will help with some spacing issues I'm seeing:
![Screenshot 2021-09-28 at 12-57-33 KoBoToolbox - RIDL](https://user-images.githubusercontent.com/200230/135074515-637f3533-49d6-440f-b435-56e0fc5122e9.png)